src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -3288,12 +3351,13 @@ private boolean loadCCSTSVFiles(IInternalContest contest, File cdpConfigDirector
         boolean loaded = false;
 
         String teamsTSVFile = cdpConfigDirectory.getAbsolutePath() + File.separator + LoadICPCTSVData.TEAMS_FILENAME;
+        String teamsTSV2File = cdpConfigDirectory.getAbsolutePath() + File.separator + LoadICPCTSVData.TEAMS2_TSV;
```

Comment:
It's a bit confusing that this variable is named `teamsTSV2File` when what it's referring to is a `teams2.tsv` file.  Why not name the variable `teams2TSVFile`?

(I can see a -- very slight -- argument that the position of the `2` in `teams2TSVFile` could be slightly misleading since use of `2` like that is commonly used to replace the word `to`; for example, `tex2pdf`,   `xml2json`, etc.  However, there's already a `2` in the existing name -- it reads like _teamsTSV to File_, which could just as easily be misinterpreted using that same logic.  I think it would be better to help avoid confusion between `teams` and `teams2` by renaming the variable.)

Reply:
Yes.  Agreed.  Changed in next commit.